src/context.ts

```diff
@@ -325,8 +372,19 @@ export default class Context {
 			}
 		}
 
-		const assignment = {
+		const assignment: ExperimentData = {
 			id: 0,
+			name: "",
```

Comment:
This is wrong, this type does not use any of these fields, and it should be a different type. This is not at all an ExperimentData, may contain some similar fields, but it is not the same. Leave it as it was, and call the type an `Assignment` containing only these fields.

Reply:
Added :)